### ISSUE TITLE
[SF][A11y][F6Navigation]: SHIFT + F6 navigation doesn't work for nested fast nav groups

### ISSUE THREAD

**Issue by 2bno1** [SAP/ui5-webcomponents#12268]
### Bug Description

`SHIFT` + `F6` does not work" in case it contains nested fast nav group elements as below:
```html
<div data-sap-ui-fastnavgroup="true">
  <div data-sap-ui-fastnavgroup="true">
    <div data-sap-ui-fastnavgroup="true">
      <Button>Button 1</Button>
    </div>
  </div>
</div>
<div data-sap-ui-fastnavgroup="true">
  <Button>Button 2</Button>
</div>
```

### Expected Behaviour

`SHIFT` + `F6` (reserved fast navigation) should work in case it contains nested fast nav group elements

### Isolated Example

https://stackblitz.com/edit/github-entmrtfv?file=src%2FApp.tsx,index.html

### Steps to Reproduce

1. Go to https://stackblitz.com/edit/github-entmrtfv?file=src%2FApp.tsx,index.html
2. Press `SHIFT` + `F6` to navigate focus through buttons
3. Focus is getting stuck on "Button 1"

### Priority

None

### UI5 Web Components Version

2.13.2

### Browser

Chrome

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by duygu-rmdn** [comment]
Hi @UI5/ui5-team-webc, could you please take a look?
